src/components/ProgramInfo.tsx

```diff
@@ -29,6 +41,23 @@ export const ProgramInfo: React.FC<ProgramDetailsProps> = ({
     programInfo,
     classInfo,
 }): JSX.Element => {
+    const { me } = useMe();
+    let fullClassInfo;
+    let availableClassInfo;
+    if (me && me.volunteer) {
+        fullClassInfo = classInfo.filter(
+            (info) => info.volunteerSpaceAvailable === 0,
+        );
+        availableClassInfo = classInfo.filter(
+            (info) => info.volunteerSpaceAvailable !== 0,
+        );
```

Comment:
you should be able to do this in one pass of the array instead of two using `forEach` or just a `for` loop like:
```
let fullClassInfo = [];
let availableClassInfo = [];
classInfo.forEach((info) => (info.volunteerSpaceAvailable === 0 ? fullClassInfo : availableClassInfo).push(info));
```

same for in the else block